tutorial_query_expansion.md

```diff
@@ -2,20 +2,21 @@
 
 ## Let's get started
 
-This tutorial will show you how to enable the query expansion feature which allows you to increase the recall for query terms with multiple results,
-especially long tail queries.
+This tutorial will show you how to enable the query expansion feature which allows you to **increase the efficiency for search** for ambiguous or long-tail* query terms. 
```

Comment:
Suggest a change to:

This tutorial shows you how to enable the query expansion feature to **increase the efficiency for search** for ambiguous or long-tail* query terms.